### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by mosuka**
- Update Cargo.toml
    - Add dependencies
- Update Makefile
    - make source CSV files in Makefile
- Update README.md
- Get CSV files using glob

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+extern crate glob;
```
Comment (johtani) [inline src/lib.rs]: Don't need extern.

File: Makefile [hunk 2]
```diff
@@ -23,8 +23,22 @@ build:
...
-lindera-ipadic-neologd: build
-	$(BIN_DIR)/lindera-ipadic-neologd $(MECAB_IPADIC_DIR) $(LINDERA_IPADIC_NEOLOGD_DIR)
+mecab-ipadic-neologd:
+ifeq ($(wildcard ./mecab-ipadic-neologd-master.zip),)
+	curl -L https://github.com/neologd/mecab-ipadic-neologd/archive/master.zip > ./mecab-ipadic-neologd-master.zip
+endif
+ifeq ($(wildcard ./mecab-ipadic-neologd-master/*),)
+	unzip mecab-ipadic-neologd-master.zip
+endif
+ifeq ($(wildcard ./mecab-ipadic-neologd-master/build/mecab-ipadic-2.7.0-20070801-neologd-20200130/*),)
```
[the remark this answers is not in the document]
Comment (mosuka) [reply Makefile]: I updated this branch. How about this?

File: Makefile [hunk 3]
```diff
@@ -1,18 +1,18 @@
 BIN_DIR ?= ./bin
-MECAB_IPADIC_DIR ?= ./mecab-ipadic-neologd
+MECAB_IPADIC_NEOLOGD_DIR ?= ./mecab-ipadic-neologd
 LINDERA_IPADIC_NEOLOGD_DIR ?= ./lindera-ipadic-neologd
-VERSION ?= 20200130
```
[the remark this answers is not in the document]
Comment (mosuka) [reply Makefile]: I updated this branch. How about this?

File: src/lib.rs (hunk 1 above)
Comment (mosuka) [reply src/lib.rs]: Deleted.

**Comment by mosuka** [comment]
`make lindera-ipadic-neologd` will produce `lindera-ipadic-<IPADIC_VERSION>-neologd-<NEOLOGD_VERSION>.tar.bz2`

**Comment by johtani** [comment]
LGTM!

**Pull request merged**